Week-9/src/App.js

```diff
@@ -0,0 +1,73 @@
+import "./App.css";
+import { useState, useEffect } from "react";
+
+function App() {
+  const [todo, setTodo] = useState([
+    { name: "Todo1", done: false },
+    { name: "Todo2", done: false },
+    { name: "Todo3", done: false },
+  ]);
+
+  const [newTask, setNewTask] = useState("");
+
+  const markdone = (index) => {
+    return () => {
+      console.log(index);
+      const temp = [...todo];
+      temp[index].done = true;
+      setTodo(temp);
+    };
+  };
+
+  const textareaChange = (props) => {
+    setNewTask(props.target.value);
+  };
+
+  const saveNewTask = () => {
+    const temp = [...todo];
+    temp.push({ name: newTask, done: false });
+    setTodo(temp);
+    setNewTask("");
+  };
+
+  return (
+    <div className="App">
+      <div>
+        <h2>Add todo</h2>
+        <div>
+          <textarea value={newTask} onChange={textareaChange}></textarea>
```

Comment:
Add todo can be one component and list of all todos can be another please sepearte these two